src/app/app-menu.ts

```diff
@@ -715,22 +716,32 @@ export class AppMenu {
           id: C2_CHANNELS_MENU_ID,
           submenu: [
             {
-              click: (_item) =>
-                windowHandler.switchClient(ClientSwitchType.CLIENT_2_0),
+              click: (_item) => {
```

Comment:
better create a dedicated function with arrow function declaration, this would help in performance issue.